labgrid/remote/client.py

```diff
@@ -1023,12 +1057,16 @@ def main():
                                       help="list available resources")
     subparser.add_argument('-a', '--acquired', action='store_true')
     subparser.add_argument('-e', '--exporter')
+    subparser.add_argument('--sort-last-changed', action='store_true',
+                           help="sort by matched place's changed date and show place and date")
```

Comment:
It's unclear which order this uses. Should that be configurable?

Reply:
Hmm, currently not. "--sort-last-changed" doesn't sound like it would do anything with places, tough.

Reply:
Right. I struggled to come up with a good name here. Maybe `--sort-by-matched-place-change`?

Reply:
I renamed the option for the resources subcommand to `--sort-by-matched-place-change` and mentioned the sort order in the help text.